ios/shapeshift.xcodeproj/project.pbxproj

```diff
@@ -38,7 +38,7 @@
 		00E356F21AD99517003FC87E /* shapeshiftTests.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = shapeshiftTests.m; sourceTree = "<group>"; };
 		0483E1162BF64354001FE65D /* Expo.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Expo.plist; sourceTree = "<group>"; };
 		0944E4151AB545FD92C041F6 /* shapeshift-Bridging-Header.h */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 4; includeInIndex = 0; lastKnownFileType = sourcecode.c.h; name = "shapeshift-Bridging-Header.h"; path = "shapeshift/shapeshift-Bridging-Header.h"; sourceTree = "<group>"; };
-		13B07F961A680F5B00A75B9A /* ShapeShift.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; name = ShapeShift.app; path = shapeshift.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		13B07F961A680F5B00A75B9A /* ShapeShift.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = ShapeShift.app; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
Is the case intended here i.e `ShapeShift.app` vs. `shapeshift.app` ?

Reply:
Honestly I think it's intended as Expo merged both `displayName` and `name` properties in the config, it seems like it's something managed by the `prebuild` command